dbt/models/default/columns.md

```diff
@@ -0,0 +1,384 @@
+## permit_address_full
+
+{% docs column_permit_address_full %}
+Address where permitted work will take place.
+
+This field combines the "mailing address" field in the source data
+with the concatenated output of the address component fields like
+`address_street_dir`, `address_street_name`, etc. We prefer the
+mailing address if it exists, and otherwise we substitute the
+concatenated address components.
+
+This field usually includes the municipality name for municipalities
+outside Chicago. If the municipality name is missing, the address
+is either in Chicago, or we generated it based on the concatenation
+of the address component fields, which do not include municipality
+name.
+{% enddocs %}
+
+## permit_assessable
+
+{% docs column_permit_assessable %}
+Whether the permitted work will affect the property's assessed
```

Comment:
**nitpick:** Passive voice + presumably if the permit is marked `Assessable` then the permit value has _already_ affected the value (i.e. been worked)?

```suggestion
Whether the permitted work affects the property's assessed
```

Reply:
Hmm, this is a good question and I'm not actually sure of the answer. I'll provisionally accept the suggestion, and I'll make sure to clear this up in my Q&A chat on Thursday.